### ISSUE TITLE
Seadragon(seajax) open specific filterbox after search

### ISSUE THREAD

**Issue by linmic30** [seajax/seajax#25]
In my website i use the seajax pivot viewer to view a large amount of images. So to not search endlessly long for the picture i need i usually use the search box in the view to filter on a specific property of my collection. Now when i press enter it works perfectly and it filters my collection. I could however be possible that multiple properties have the same value so to further filter my search i use the filters beneath the search box. This also works fine.

Now for my question: Is it possible to open such a filterbox after i press enter, so that the specific values for a certain property are shown?

I already took a look at it and i belief that i need to do it in the pivot viewer js, however i cannot find the right way to do it. I could not find anything on stackoverflow about it.

Thank you in advance.

**Comment by sethbrenith** [comment]
Sorry, PivotView.js doesn't provide very many relevant hooks for doing this kind of thing, but I think you can still get the effect you want by hooking into UI events directly. For instance, try running the following in the dev console on [seajax.github.io](http://seajax.github.io/), and then do a search. This code listens for `submit` events on the search box or `mousedown` in the suggestion list and then simulates a click on a particular facet category label to open it.

    function openProductionYear() {
        Array.prototype.slice.call(document.querySelectorAll(".pivot_facetlabel")).find(function (element) {
            return element.textContent === "Production Year";
        }).click();
    }
    document.querySelector("form.pivot_watermark").addEventListener("submit", openProductionYear, false);
    document.querySelector(".pivot_results").addEventListener("mousedown", openProductionYear, false);
